<ui version="4.0" >
 <class>UnreferredDocumentsWidget</class>
 <widget class="QWidget" name="UnreferredDocumentsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Unreferred DocumentsWidget</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Search</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="textlabel_url" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Base Folder:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
          </property>
          <property name="margin" >
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLSHistoryCombo" name="baseDirCombo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Ignored" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="acceptDrops" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pickUrlButton" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>2</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Unreferred Documents</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="KLineEdit" name="filterLineEdit" >
        <property name="clickMessage" >
         <string>Filter Documents...</string>
        </property>
        <property name="showClearButton" stdset="0" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="progressLabel" >
       <property name="minimumSize" >
        <size>
         <width>94</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Ready</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="textVisible" >
        <bool>false</bool>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>1</number>
       </property>
       <property name="topMargin" >
        <number>1</number>
       </property>
       <property name="rightMargin" >
        <number>1</number>
       </property>
       <property name="bottomMargin" >
        <number>1</number>
       </property>
       <property name="horizontalSpacing" >
        <number>0</number>
       </property>
       <property name="verticalSpacing" >
        <number>0</number>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" name="elapsedTimeLabel" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>hh:mm:ss</string>
         </property>
         <property name="text" >
          <string>Elapsed time: </string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="indent" >
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="elapsedTimeValueLabel" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>hh:mm:ss</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="indent" >
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLSHistoryCombo</class>
   <extends>KHistoryComboBox</extends>
   <header>klshistorycombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
